web/src/components/HeaderLogo.tsx

```diff
@@ -52,7 +51,7 @@ export const HeaderLogo = ({ link }: PropsType): ReactElement => {
   return (
     <LogoContainer>
       <Link to={link}>
-        <img src={src} alt={appName} />
+        <img src={src} alt={appName} width='100%' height='auto' />
```

Comment:
whats this about?

Reply:
the problem is that the image gets rendered with 0px then browser requests resources and rerenders image with correct dimensions.
[Explicit Dimensions](https://dev.to/inhuofficial/quick-tips-how-to-fix-image-elements-do-not-have-explicit-width-and-height-in-page-speed-insights-lighthouse-3776)